pkg/apis/apps/v1beta1/register.go

```diff
@@ -41,6 +41,10 @@ var (
 // Adds the list of known types to api.Scheme.
 func addKnownTypes(scheme *runtime.Scheme) error {
 	scheme.AddKnownTypes(SchemeGroupVersion,
+		&Deployment{},
+		&DeploymentList{},
+		&DeploymentRollback{},
+		&Scale{},
```

Comment:
Yeah, but there's a separate issue to solve it, can't recall from the top of my head.